tests/test_transformers.py

```diff
@@ -44,16 +43,19 @@ def test_apply_node_property_classes(assert_same_elements):
             "data": {
                 "ujt_id": node_name,
             },
+            "classes": "",
```

Comment:
Previously, some transformers overwrote this field `element["classes"] = "my_class"`. Now, they append to it, so this field needs to be present in the input data.